### DISCUSSION TITLE
Dimensity

### DISCUSSION THREAD

**Discussion by Jordyjj** [skyline-emu/skyline#2080] [Q&A] [answered]
Hola buenas noches, soy nuevo por aquí pero tengo tiempo siguiendo su proyecto y me parece un muy buen trabajo el que están haciendo, de verdad 20 de 20 por todo. Pero tengo una pregunta. Cuando podrán sacar una versión de Skyline Edge donde puedan los dispositivos con un procesador Dimensity optar por aplicarle Drivers para un mejor rendimiento ya que en mi poco X3 GT la opción no me aparece disponible aún. Porfavor se los agradecería muchísimo si pudieran hacer que esto sea posible. Muchas gracias por su trabajo. Espero pronta y positivas respuestas.

**Comment by JERD001** [comment, marked as answer]
No tendrá en un buen tiempo, no existen drivers aparte de los que provee el fabricante y así se quedará por un largo tiempo si es que salen algún día.

**Reply by Jordyjj** [reply]
Sería muy bueno que los colocarán

**Reply by Crytonics** [reply]
Right now you can only use system drivers for mali gpu-s.
